testC615CloseRed_edwardkort fails after fix for #641

```
com.shootoff.camera.TestCameraManagerUserProvided > testC615CloseRed_edwardkort FAILED

    java.lang.AssertionError: There was 1 shot that was detected that isn't account for: (320.90, 75.21, RED)

    Expected: <true>

         but: was <false>

        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)

        at org.junit.Assert.assertThat(Assert.java:956)

        at org.junit.rules.ErrorCollector$1.call(ErrorCollector.java:65)

        at org.junit.rules.ErrorCollector.checkSucceeds(ErrorCollector.java:78)

        at org.junit.rules.ErrorCollector.checkThat(ErrorCollector.java:63)

        at com.shootoff.camera.ShotDetectionTestor.checkShots(ShotDetectionTestor.java:112)

        at com.shootoff.camera.TestCameraManagerUserProvided.testC615CloseRed_edwardkort(TestCameraManagerUserProvided.java:83)
```

I suppose the fix, which cut out about 115 ms on the first shot, is enough to let a dupe through in this case.

May be worth just accepting the dupe because it's a terrible trigger pull.

--- a/src/main/java/com/shootoff/camera/CameraManager.java
+++ b/src/main/java/com/shootoff/camera/CameraManager.java
@@ -257,7 +257,7 @@ public class CameraManager implements ObservableCloseable, CameraEventListener, 
 
 	// Used by click-to-shoot and tests to inject a shot via the shot detector
 	public void injectShot(ShotColor color, double x, double y, boolean scaleShot) {
-		shotDetector.addShot(color, x, y, getCurrentFrameTimestamp(), scaleShot);
+		shotDetector.addShot(color, x, y, System.currentTimeMillis(), scaleShot);
 	}
 
 	public void clearShots() {
@@ -265,14 +265,14 @@ public class CameraManager implements ObservableCloseable, CameraEventListener, 
 	}
 
 	public void reset() {
-		resetStartTime();
+		resetStartTime(0);
 		shotDetector.reset();
 		deduplicationProcessor.reset();
 		cameraView.reset();
 	}
 
-	private void resetStartTime() {
-		startTime = System.currentTimeMillis();
+	private void resetStartTime(long timestamp) {
+		startTime = timestamp;
 	}
 
 	@Override
@@ -779,7 +779,7 @@ public class CameraManager implements ObservableCloseable, CameraEventListener, 
 
 	public long getCurrentFrameTimestamp() {
 		if (startTime == 0) {
-			resetStartTime();
+			resetStartTime(System.currentTimeMillis());
 			return 0;
 		}
 
@@ -788,7 +788,7 @@ public class CameraManager implements ObservableCloseable, CameraEventListener, 
 
 	public long cameraTimeToShotTime(long timestamp) {
 		if (startTime == 0) {
-			resetStartTime();
+			resetStartTime(timestamp);
 			return 0;
 		}
 

--- a/src/main/java/com/shootoff/gui/CanvasManager.java
+++ b/src/main/java/com/shootoff/gui/CanvasManager.java
@@ -161,7 +161,7 @@ public class CanvasManager implements CameraView {
 				// addArenaShot when they go through the arena camera feed's
 				// canvas manager
 				if (this instanceof MirroredCanvasManager) {
-					final long shotTimestamp = cameraManager == null ? 0 : cameraManager.getCurrentFrameTimestamp();
+					final long shotTimestamp = System.currentTimeMillis();
 
 					addShot(new Shot(shotColor, event.getX(), event.getY(), shotTimestamp, config.getMarkerRadius()),
 							false);

